censusdis/symbolic.py

```diff
@@ -0,0 +1,95 @@
+# Copyright (c) 2022 Darren Erik Vengroff
+
+"""
+Utilities for creating symbolic names.
+
+This module processes data sets from the US Census
+and their respective symbolic names for
+documentation purposes.
+"""
+
+import os
+import censusdis.data as ced
+import censusdis.datasets
+
+class symbolic:
+    def __init__(self):
+        self.dictionary = {}
+        self.module_message = [
+            "# Copyright (c) 2022 Darren Erik Vengroff\n",'"""\n'
+            "This module contains abbreviated names for commonly used data sets.\n",
+            "These are typically used as the first argument to :py:func:`censudis.data.download`.\n",
+            "There are a lot more data sets available than there are symbolic names here.\n",
+            "But you can always use raw strings. For example, even for `ACS5` you can use\n",
+            "`acs/acs5` instead.\n", '"""\n' 
+        ]
+
+    def store_dataset(self, dataset_list: list, url_list: list):
+        for item in dataset_list:
+            if item not in self.dictionary.values():
+                index = dataset_list.index(item)
```

Comment:
Instead of getting the index of the item here and then grabbing the link from the same location, I think you could have just zipped the two lists to begin with. So, three lines above, replace

```
for item in dataset_list:
```

with 

```
for item, link in zip(dataset_list, url_list)
```

This would save you having to scan the list to get the index every time.